docs/DreamBig/Sprint 2 documentation/Diagrams/UML.md

```diff
@@ -0,0 +1,9 @@
+# DreamBig UML Diagram
+
+(Subject to change)
+
+![UML Diagram](../../assests/images/uml_diagram.jpg)
+
+[Miro Link](https://miro.com/app/board/uXjVO5LY6zo=/?share_link_id=917467665424)
+
+    Password: DreamBig
```

Comment:
**issue:** Could we please avoid sharing passwords in Git repositories 🙏🏼? All repositories are public and everyone can see it 😱